app/client/jsx/Map.jsx

```diff
@@ -55,35 +56,52 @@ class Map extends Component {
 
     getGlobalStats() {
         const users = this.props.users
-        
+
         const numInHouse = _.sumBy(Object.keys(users), room => users[room].length)
         const numInHallways = (users.hallway || []).length
-        const numInBathrooms = _.sumBy(
-            Object.keys(users).filter(
-                room => room.endsWith('bathroom')
-            ),
-            room => users[room].length
-        )
-        const numInRooms = numInHouse - numInHallways - numInBathrooms
+        const numInRooms = numInHouse - numInHallways
+
+        const { stats: config } = Config.map
+
+        const mapStats = [
+            {
+                key: 'numberInHouse',
+                count: numInHouse
+            },
+            {
+                key: 'numberInRooms',
+                count: numInRooms
+            },
+            {
+                key: 'numberInHallways',
+                count: numInHallways
+            }
+        ]
+        .filter(({ key }) => config[key].display)
+        .map(({ key, count }) => ({
+            text: config[key].text.replace('{0}', count),
+            emoji: config[key].emoji
+        }))
+
+        config.additionalStats.forEach(({ text, match, emoji }) => {
+            const rooms = _.isArray(match) ?
+                _.intersection(Object.keys(users), match) :
+                Object.keys(users).filter(room => new RegExp(match).test(room))
+            const count = _.sumBy(rooms, room => users[room].length)
+            mapStats.push({
+                text: text.replace('{0}', count),
```

Comment:
FYI if we're not doing multiple format parameters, `{0}` might be a little misleading

Reply:
Agreed...I tried `{}` but it looked a little weird to me. There must be a standard for this. Do you have a suggestion? Maybe `%` or `%s`? Wildcard?

Reply:
what about `{count}`?
`"{count} taking bathroom break"`
`text.format(count=count)`